Source/Common/IGListIndexSetResult.h

```diff
@@ -36,14 +36,14 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  The moves from an index in the old collection to an index in the new collection.
  */
-@property (nonatomic, copy, readonly) NSArray<IGListMoveIndex *> *moves;
+@property (nonatomic, strong, readonly) NSArray<IGListMoveIndex *> *moves;
 
 /**
  Returns whether the result has any changes or not.
 
  @return `YES` if the result has changes, `NO` otherwise.
  */
-- (BOOL)hasChanges;
+@property (nonatomic, assign, readonly) BOOL hasChanges;
```

Comment:
This is a breaking change. We should probably revert this and handle #402 as a separate PR. Will require a changelog entry, etc.